apps/aecore/src/aec_keys.erl

```diff
@@ -296,3 +306,11 @@ to_local_dir(NewFile, Bin) ->
 
 from_local_dir(NewFile) ->
     file:read_file(NewFile).
+
+
+-spec check_keys_pair(binary(), binary(), atom(), atom(), atom()) -> boolean().
+check_keys_pair(PubKey, PrivKey, Algo, Digest, Curve) ->
+    SampleMsg = <<"random message">>,
```

Comment:
wasn't there standard verify string for keys? "Sanity message"? This is good, but if you run into it in the future, please adjust.